### PR TITLE
docs(install): update Helm chart install

### PR DESCRIPTION
**Pull request by aimeeu**
Resolves Jira: [RUBKIK-318]

I also restructured the Agent quickstart into an install directory.

The Helm install section is based on the chart repo [README](https://github.com/armory-io/agent-k8s/tree/master/deploy/charts/armory-k8s-full).  

Deploy preview:  https://deploy-preview-1004--armory-docs.netlify.app/docs/armory-agent/armory-agent-install/install-agent-service-helm/
New content files:
- content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/frag-helm-agent-mode.md
- content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/frag-helm-infra-mode.md
- content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/index.md

### CODE REVIEW

File: content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/index.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+With the file, you can avoid setting individual `env` parameters in the `helm install` command. Instead include the `--values` parameter as part of the Helm install command:
+
+```bash
+--values=<path>/values.yaml
+```
+
+## Set a custom Agent image registry
+
+If you need to store the Armory Agent image in a different registry, you can specifiy that repository using the following settings:
```
Comment (danielGz) [inline content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/index.md]: I would say rather: "if you need to download the image from a different repository".

File: content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/index.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+You can update configuration options via the command line by using `--set config.<option>`. For example, if you want to set `logging.level` to `debug`, use `--set config.logging.level=debug` to do that.
+
+
+### Set configuration in a file
+
+You can also override default settings in your own `armory-agent.yml` file. For example, if you want to modify the default logging settings, create an `armory-agent.yml` file with the following content:
+
+{{< prism lang="yaml" >}}
+config:
```
Comment (danielGz) [inline content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/index.md]: when using agentyml the format is the same as the kubectl install method, there is no config prefix.

File: content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/index.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+--namespace=dev \
+--set hub.auth.armory.clientId=clientID123,hub.auth.armory.secret=s3cret \
+,config.logging.level=debug,config.kubernetes.retries.maxRetries=5 \
+,config.kubernetes.retries.backOffMs=5000,config.prometheus.enabled=true
+{{< /prism >}}
+
+The same custom configuration in an `armory-agent.yml` file:
+
+{{< prism lang="yaml" >}}
+config:
```
Comment (danielGz) [inline content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/index.md]: same as before, there is no `config` nesting.

File: content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/frag-helm-infra-mode.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+   You must set your **kubeconfig** file and secret.
+
+   {{< prism lang="bash" line="4" >}}
+   helm install armory-agent armory-charts/agent-k8s-full \
+   --create-namespace \
+   --namespace=<agent-namespace> \
+   --set hub.auth.armory.clientId=<your-clientID> \
+   ,hub.auth.armory.secret=<your-clientSecret> \
+   ,kubeconfigs.<account-name>.file=<kubeconfig> \
```
Comment (danielGz) [inline content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/frag-helm-infra-mode.md]: let's specify somewhere that this is the name of the file used when creating the secret.  E.g. if the secret was created with `--from-file=<path>/.kube/config` this value should be **config**

File: content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/frag-helm-infra-mode.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+   You must include your gPRC endpoint, such as `localhost:9090`.
+
+   You must set your **kubeconfig** file and secret.
+
+   {{< prism lang="bash" line="4" >}}
+   helm install armory-agent armory-charts/agent-k8s-full \
+   --create-namespace \
+   --namespace=<agent-namespace> \
+   --set config.clouddriver.grpc=<endpoint> \
+   ,kubeconfigs.<account-name>.file=<kubeconfig> \
```
Comment (danielGz) [inline content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/frag-helm-infra-mode.md]: same as before

**Review by danielGz — COMMENTED** [review]
left some minor comments :)

**Review by 317brian — APPROVED** [review]
LGTM. One really minor nitpick about not having intro sentences for the lists in the chart overview and prereq sections on the helm install page, but that's personal preference

**Comment by aimeeu** [comment]
> 

That's a great point and very valid from an SEO perspective!

**Pull request merged**